src/validator.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable max-len */
+const errors = {
+  textIsMissing: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  toCaseIsMissing: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  toCaseIsNotSupported: 'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+};
+
+const validCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+function validator(text, toCase) {
+  return {
+    errors: [
+      ...(!text ? [{ message: errors.textIsMissing }] : []),
+      ...(!toCase ? [{ message: errors.toCaseIsMissing }] : []),
+      ...(!validCases.includes(toCase) && toCase ? [{ message: errors.toCaseIsNotSupported }] : []),
+    ],
+  };
```

Comment:
this feels like something from codewars. not sure if the code like that has a place in a real project

Reply:
agree, It's better to make this code more readable